R/pkg/R/DataFrame.R

```diff
@@ -2635,7 +2635,7 @@ setMethod("write.df",
             write <- callJMethod(df@sdf, "write")
             write <- callJMethod(write, "format", source)
             write <- callJMethod(write, "mode", jmode)
-            write <- callJMethod(write, "save", path)
+            write <- callJMethod(write, "save", options)
```

Comment:
I am not sure I see a `save` method which takes in `options`. What if we just call `write <- callJMethod(write, "options", options)` in the line before `write <- callJMethod(write, "save", path)` ?